File: 2500-delete-greatest-value-in-each-row/2500-delete-greatest-value-in-each-row.py
class Solution:
    def deleteGreatestValue(self, grid: List[List[int]]) -> int:
        for row in grid:
            row.sort()
        
        ans = 0        
        for i in range(len(grid[0])):
            tmp = []
            for j in range(len(grid)):
                tmp.append(grid[j].pop()) 
            ans += max(tmp)  
        
        return ans

            

File: 2500-delete-greatest-value-in-each-row/2500-delete-greatest-value-in-each-row.go
func deleteGreatestValue(grid [][]int) int {
    for i := range grid {
        sort.Ints(grid[i])
    }    
    ans := 0
    for i := 0; i < len(grid[0]); i++ {
        max := grid[0][i]
        for _, row := range grid[1:] {
            if row[i] > max {
                max = row[i]
            }
        }
        ans += max
    }
    return ans
}
